### PR TITLE
disable cat pre&post transpose

### PR DESCRIPTION
**Pull request by shuhand0**
mps cat out tensor is always contiguous.

### CODE REVIEW

File: aten/src/ATen/native/mps/operations/Shape.mm [hunk 1]
```diff
@@ -192,6 +192,9 @@ void check_shape_except_dim(const Tensor &first, const Tensor &second,
        MemoryFormat memory_format,
        const Tensor& out) {
   using namespace mps;
+
+  memory_format = MemoryFormat::Contiguous;
```
Comment (razarmehr) [inline aten/src/ATen/native/mps/operations/Shape.mm:196]: How does this fix the issue? Does the test_mps pass with this change?
If you want memory_format to be contiguous, then why don't you pass the input tensor as contiguous?

**Review by razarmehr — CHANGES_REQUESTED** [review]

File: aten/src/ATen/native/mps/operations/Shape.mm (hunk 1 above)
Comment (kulinseth) [reply aten/src/ATen/native/mps/operations/Shape.mm:196]: @razarmehr , this is similar to what Denis is doing in the case of elu_backward. We need to make sure the NHWC case is handled. I think we should centralize the contiguous call and any copy which needs to happen.

Comment (shuhand0) [reply aten/src/ATen/native/mps/operations/Shape.mm:196]: It passes test_mps locally. The ACDC case passes after the fix.  I'll work on a unit test that is smaller enough to show the case.

Comment (razarmehr) [reply aten/src/ATen/native/mps/operations/Shape.mm:196]: OK, so we're bypassing the NHWC fix that was committed before by explicitly making the tensors contiguous before doing cat. If you think this won't affect performance much, we can go with it to unblock us, and later come up with a proper fix.

**Review by kulinseth — APPROVED** [review]

**Comment by kulinseth** [comment]
```
======================================================================
FAIL [0.140s]: test_output_match_triangular_solve_cpu_float32 (__main__.TestConsistencyCPU)
----------------------------------------------------------------------
```
This is fixed in other commit.

**Pull request merged**